src/entities/item/ui/CardItemNavigator.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from "classnames"
+
+import { BasicButton } from "~/shared/ui"
+import { useCustomTranslation } from "~/shared/utils"
+
+interface CardItemNavigatorProps {
+  isBackButtonShown?: boolean
+  onBackButtonClick?: () => void
+
+  isSubmitButtonShown?: boolean
+  isNextButtonShown?: boolean
+  onNextButtonClick?: () => void
+
+  isSkippable?: boolean
+  isNextButtonDisable?: boolean
+}
+
+export const CardItemNavigator = ({
```

Comment:
You do not need this component. Just use BasicButton component where is makes sense

Reply:
Done